src/main/java/seedu/address/model/TbmManager.java

```diff
@@ -23,7 +23,7 @@
 public class TbmManager implements ReadOnlyTbmManager {
 
     private final UniqueClientList clients;
-    private final UniqueTagSet tags;
+    private final UniqueTagSet tags; // todo: deprecate use of UniqueTagSet
```

Comment:
can't we already remove it? Since its not being used anywhere in the whole codebase

Reply:
I'm adding this as a separate issue for an upcoming PR so merging these bug fixes can be expedited.